cms/templates/base.html

```diff
@@ -5,13 +5,14 @@
 from django.utils.translation import get_language_bidi
 from django.template.defaultfilters import escapejs
 import json
-
-dir_rtl = 'rtl' if get_language_bidi() else 'ltr'
 %>
+<%def name="dir_rtl()">
+  <% return 'rtl' if get_language_bidi() else 'ltr' %>
+</%def>
```

Comment:
Honestly, I don't know the reasons for preferring one over the other.  Probably we should move the `<%def` into a separate Mako file, and import into the places we need it.